gc/base/Collector.cpp

```diff
@@ -192,6 +196,10 @@ MM_Collector::preCollect(MM_EnvironmentBase* env, MM_MemorySubSpace* subSpace, M
 {
 	MM_GCExtensionsBase* extensions = env->getExtensions();
 
+#if defined(OMR_GC_CONCURRENT_SCAVENGER)
+	completeConcurrentScavenge(env);
```

Comment:
@amicic should this be on GlobalCollector instead?  Isn't the Scavenger also a sub class of collector?